katas/descendingorder/adriel/Program.cs

```diff
@@ -0,0 +1,16 @@
+namespace DescendingOrder.Adriel;
+
+public class DescendingOrder {
+    public int getDescendingOrder(int number)
```

Comment:
For C#, normally PascalCase is the rule for naming class members that are public:
https://learn.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/coding-conventions#naming-conventions